internal/cephfs/controllerserver.go

```diff
@@ -1067,7 +1067,7 @@ func (cs *ControllerServer) DeleteSnapshot(
 	if acquired := cs.SnapshotLocks.TryAcquire(sid.RequestName); !acquired {
 		log.ErrorLog(ctx, util.SnapshotOperationAlreadyExistsFmt, sid.RequestName)
 
-		return nil, status.Errorf(codes.Aborted, util.VolumeOperationAlreadyExistsFmt, sid.RequestName)
+		return nil, status.Errorf(codes.Aborted, util.SnapshotOperationAlreadyExistsFmt, sid.RequestName)
```

Comment:
can you please fix the same for groupsnapshot as well?

Reply:
we are taking volumelocks for it, please look into it as need to take snapshot locks. please take care of this as a new PR.